addons/mail/static/tests/helpers/mock_server.js

```diff
@@ -744,6 +748,62 @@ MockServer.include({
         this._widget.call('bus_service', 'trigger', 'notification', [notification]);
         return channelInfo;
     },
+    /**
+     * Simulates the `channel_seen` method of mail.channel.
+     *
+     * @private
+     * @param {integer[]} channel_ids
+     * @param {integer} last_message_id
+     */
+    async _mockMailChannelChannelSeen(channel_ids, last_message_id) {
+        // Update record
+        const channel_id = channel_ids[0];
+        if (!channel_id) {
+            return;
+        }
+        const messagesBeforeGivenLastMessage = this._getRecords('mail.message', [
+            ['channel_ids', 'in', [channel_id]],
+            ['id', '<=', last_message_id]
+        ]);
+        if (!messagesBeforeGivenLastMessage || messagesBeforeGivenLastMessage.length === 0) {
+            return;
+        }
+        const domain = [
+            '&',
+            [['channel_id', 'in', [channel_id]], ['partner_id', '=', this.currentPartnerId]],
+            ['|', ['seen_message_id', '=', false], ['seen_message_id', '<', last_message_id]],
+        ];
+        const channelPartner = this._getRecords('mail.channel.partner', domain)[0];
+        if (!channelPartner) {
+            // FIXME    not sure
+            this._mockCreate('mail.channel.partner', {
+                channel_id: channel_id,
+                fetched_message_id: last_message_id,
+                seen_message_id: last_message_id,
+            });
+        } else {
+            this._mockWrite('mail.channel.partner', [channelPartner.id], {
```

Comment:
wrong format, there is the issue currently on other mocks.

The 2nd argument of write should be an array

Reply:
`[[ids], data]`